Helm-388: Rename Helm in the Helm docs (#653)
OpenNMS/grafana-plugin: docs/modules/getting_started/pages/migrating_from_opennms_datasource.adoc

> = Migrate From OpenNMS Data Source Plugin

Reviewer: ```suggestion
= Migrate from OpenNMS Data Source Plugin
```

* A configured OpenNMS datasource.
* One or more dashboards using the OpenNMS datasource.

> == Install the plugin

Reviewer: This is me being pedantic, since we're talking about two plugins. :)

```suggestion
== Install the {page-component-title}
```

Before completing the migration, ensure that you have xref:installation:requirements.adoc[installed the plugin] on your Grafana instance.

NOTE: Both the OpenNMS Data Source plugin and the {page-component-title} can be installed on your Grafana instance while you complete the migration.

== Grafana setup

After you have installed the {page-component-title}, you must enable it in Grafana to make the provided panels and datasources available:

. Navigate to the home page of your Grafana instance.
. In the left navigation menu, click menu:Configuration[Plugins]:
+
image::gf-plugins.png["Grafana menu displaying navigation options. The Plugins option is highlighted.", 250]

> . Select the *OpenNMS plugin for Grafana*.

Reviewer: ```suggestion
. Select *OpenNMS Plugin for Grafana*.
```

** If the plugin is not listed, try restarting your Grafana instance.

> ** If the issue persists, make sure the plugin is in the Grafana plugins directory (default: `var/lib/grafana/plugins`).

Reviewer: ```suggestion
** If the issue persists, make sure the plugin files are in the Grafana plugins directory (default: `var/lib/grafana/plugins`).
```

. Click menu:Config[Enable].
+
NOTE: The Disable option indicates that the OpenNMS {page-component-title} is already enabled.
You can also choose *Update* to download the latest version of the plugin.

== Migrate datasources

The plugin provides two datasource types; the OpenNMS Performance type is equivalent to the previous "OpenNMS" datasource.

> 

Reviewer: ```suggestion
```

> The plugin provides two datasource types; the OpenNMS Performance type is equivalent to the previous "OpenNMS" datasource.

Reviewer: I think we could move this line up to be the start of the previous paragraph. 🤔
